go.mod

```diff
@@ -42,15 +41,15 @@ require (
 	github.com/gorilla/mux v1.8.1
 	github.com/rancher/dynamiclistener v0.6.0
 	github.com/rancher/lasso v0.0.0-20240705194423-b2a060d103c1
-	github.com/rancher/rancher/pkg/apis v0.0.0-20240710151157-3a6de11e4de9
+	github.com/rancher/rancher/pkg/apis v0.0.0-20240816071246-0c3b39a95054
```

Comment:
Near as I can tell, this is from release/v2.9. Since this branch is from main, this should point to the rancher/rancher branch for main, not release/v2.9.

Reply:
Good point. I had that version pulled in, didn't realize it was off that branch.

Reply:
Fixed